jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true
+  },
+  "exclude": ["node_modules", "public", ".cache"]
+}
```

Comment:
not sure why we need `jsconfig.json`  is this file being used by `vsce` while packaging?